### DISCUSSION TITLE
h2o package

### DISCUSSION THREAD

**Discussion by fsebastian93** [PaddlePaddle/PaddleSpatial#148] [BaiduKDDCup2022] [answered]
If it is possible to install h2o package in base environment. Thank you
pip install -f http://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o

**Comment by ramber1836** [comment, marked as answer]
Could you explain the necessity of using h2o. 
As I referred to the package description, I don't think we have to use Hadoop to handle the dataset we adopted? 
<img width="863" alt="image" src="https://user-images.githubusercontent.com/19561583/172083919-6f95d800-07f0-4821-8797-67735348809c.png">

**Reply by fersebas** [reply]
Hello! Our models use this boosting https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/gbm.html it permits some distribution of the data ("gaussian"). Not all gradient boosting predict the same values...

Thank you.

**Reply by ramber1836** [reply]
The package has already been added to the base env.
